src/models/db/account.js

```diff
@@ -69,6 +67,8 @@ class Account extends Model {
   }
 
   static convertFromPersistent (data) {
+    data.is_disabled = Boolean(data.is_disabled)
```

Comment:
A comment would be helpful. Also, why convert to a number below (in `convertToPersistent`)?